src/components/Tool/Home/ToolsListHome.ts

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
This looks pretty similar to what's in `ToolsListNav.ts`. Would it be possible to generate this file from `ToolsListNav.ts`? If that's not feasible, that's OK.

Reply:
Oh did you duplicate it to get a specific order for the tools? It might be OK then.

Reply:
I think you can keep it.